internal/super/project.go

```diff
@@ -58,12 +58,13 @@ func newProject(
 }
 
 type project struct {
-	service        *flowkit.Account
-	services       *services.Services
-	state          *flowkit.State
-	readerWriter   flowkit.ReaderWriter
-	projectFiles   *projectFiles
-	pathNameLookup map[string]string
+	service                *flowkit.Account
+	services               *services.Services
+	state                  *flowkit.State
+	readerWriter           flowkit.ReaderWriter
+	projectFiles           *projectFiles
+	pathNameLookup         map[string]string
+	superGeneratedAccounts []string // accounts created by "flow dev" super commands and not developer
```

Comment:
Since this is only accessible withing the dev command we can avoid adding super in the name, otherwise we should put it on all other vars, so I just suggest changing to something like createdAccounts, I would even say you can just make it list of accounts types, but that's more subjective.
```suggestion
	createdAccounts []*flowkit.Account 
```